src/lib/y2users/user.rb

```diff
@@ -145,22 +150,6 @@ def uid?
       !(uid.nil? || uid == "")
     end
 
-    # Whether two users are equal
-    #
-    # Only relevant attributes are compared. For example, the config in which the user is attached
```

Comment:
I wouldn't remove the comment mentioning than the internal id and the config were intentionally left out.